api/app/src/main/kotlin/packit/security/provider/TokenProvider.kt

```diff
@@ -23,23 +39,59 @@ class TokenProvider(val config: AppConfig) : JwtIssuer
         const val TOKEN_AUDIENCE = "packit"
     }
 
-    override fun issue(userPrincipal: UserPrincipal): String
-    {
+    inner class Builder(val userPrincipal: UserPrincipal) : JwtBuilder {
```

Comment:
the inner class stuff makes a bit messy... could move inner class outside and just pass along the config when calling instantiating class

Reply:
Done